static/app/gettingStartedDocs/apple/apple-macos.tsx

```diff
@@ -7,9 +7,10 @@ import {t, tct} from 'sentry/locale';
 // Configuration Start
 export const steps = ({
   dsn,
-}: {
-  dsn?: string;
-} = {}): LayoutProps['steps'] => [
+  sourcePackageRegistries,
+}: Partial<
+  Pick<ModuleProps, 'dsn' | 'sourcePackageRegistries'>
+> = {}): LayoutProps['steps'] => [
```

Comment:
I wish we could simplify this. It's quite clunky, especially because we repeat it in lots of places.

Reply:
yes, I talked to Arthur about it too... we should definitely simplify this, but would prefer to do that in a separate PR